frontend/packages/pipelines-plugin/locales/en/pipelines-plugin.json

```diff
@@ -12,7 +12,7 @@
   "Namespace": "Namespace",
   "Type": "Type",
   "Created": "Created",
-  "Pipeline run details": "Pipeline run details",
+  "Pipeline Run details": "Pipeline Run details",
```

Comment:
"Pipeline Run Details" (ticket description)

Reply:
Actually, it might _not_ be true? Perhaps a mistype by @bgliwa01

Reply:
The admin pages use a lower case `details`.

Reply:
It should be "Pipeline Run details"